### PR TITLE
fetch Food categories from api (categoriesFood page)

### PR DESCRIPTION
**Pull request by IsraaHamdi**
RELATES#37

### CODE REVIEW

**Review by MohammedDahalan — APPROVED** [review]

**Review by muhannaAtalla20 — APPROVED** [review]

File: otherPages/categories/categories.css [hunk 1]
```diff
@@ -1,131 +1,103 @@
...
+.header i {
+  font-size: 2.375rem;
+  color: #f79259;
+}
+.header .logo {
+  width: 4.25rem;
+  height: 3.8rem;
+}
+.title {
+  font-family: Righteous-Regular;
```
Comment (yaqootturman) [inline otherPages/categories/categories.css:28]: I like the font family here but if you want to keep it, you should change the font family for other headers on other pages

File: js/categoryFood.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+let catMeal=document.getElementsByClassName('categories-food')
+fetch('https://www.themealdb.com/api/json/v1/1/categories.php')
```
Comment (yaqootturman) [inline js/categoryFood.js]: You set local storage but I can't see where you use the data from it? so maybe you can check the local storage before fetch and if it is empty: fetch data from API, if not and it fully, get the data from local storage..
BTW: this way will reduce the time for your code

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: otherPages/categories/categories.css (hunk 1 above)
Comment (IsraaHamdi) [reply otherPages/categories/categories.css:28]: yes, In all Pages using this font

File: js/categoryFood.js (hunk 2 above)
Comment (IsraaHamdi) [reply js/categoryFood.js]: done  :)

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**